lib/restaurant.rb

```diff
@@ -0,0 +1,21 @@
+require './lib/sendtext'
+
+class Restaurant
+  attr_reader :menu, :sendtext
+
+  def initialize(sendtext = SendText.new)
+    @menu = {"Chips" => 0.89, "Chicken Tikka" => 3.99, "Burger" => 2.99}
+    @sendtext = sendtext
+  end
+
+  def read_menu
+    menu.map {|item, price| "#{item} - #{format('£%.2f', price)}"}
+  end
+
+  def complete_order(total)
+    dly_time = (Time.now + 3600).strftime('%H:%M')
+    msg="Order will be delivered by #{dly_time}. Total: #{format('£%.2f', total)}"
```

Comment:
Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)